openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -42,28 +62,40 @@ export class CourseOutline {  // eslint-disable-line import/prefer-default-expor
 
           sections.forEach(section => section.addEventListener('click', (event) => {
             const sectionToggleButton = event.currentTarget;
-            const $toggleButtonChevron = $(sectionToggleButton).children('.fa-chevron-right');
-
             if (sectionToggleButton.classList.contains('accordion-trigger')) {
               const isExpanded = sectionToggleButton.getAttribute('aria-expanded') === 'true';
-              const $contentPanel = $(document.getElementById(sectionToggleButton.getAttribute('aria-controls')));
-
               if (!isExpanded) {
-                $contentPanel.slideDown();
-                $contentPanel.removeClass('is-hidden');
-                $toggleButtonChevron.addClass('fa-rotate-90');
-                sectionToggleButton.setAttribute('aria-expanded', 'true');
+                  expandSection(sectionToggleButton);
               } else if (isExpanded) {
-                $contentPanel.slideUp();
-                $contentPanel.addClass('is-hidden');
-                $toggleButtonChevron.removeClass('fa-rotate-90');
-                sectionToggleButton.setAttribute('aria-expanded', 'false');
+                  collapseSection(sectionToggleButton);
               }
-
               event.stopImmediatePropagation();
             }
           }));
         });
+
+      const toggleAllButton = document.querySelector('#expand-collapse-outline-all-button');
+      const toggleAllSpan = document.querySelector('#expand-collapse-outline-all-span');
+      toggleAllButton.addEventListener('click', (event) => {
+          const toggleAllExpanded = toggleAllButton.getAttribute('aria-expanded') == 'true';
```

Comment:
If you wanted, you could turn this line and line 66 in the same file (```const isExpanded = sectionToggleButton.getAttribute('aria-expanded') === 'true';```) into a function, but I'm not sure it would actually be cleaner.

Reply:
Oh, you also probably want a triple ===

Reply:
Good point, I do want triple equal.